modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/utils/constants/columns-definitions/KeyTypeColumn.js

```diff
@@ -9,15 +9,12 @@
  * distribution rights of the Software.
  */
 
-const VIRTUAL_CLUSTER = 'virtual-cluster';
+import {hasVirtualCluster} from '..';
 
 const KeyTypeColumn = ({activationKey, assetsPath}) => {
-	const hasVirtualCluster =
-		activationKey.licenseEntryType === VIRTUAL_CLUSTER;
-
 	return (
 		<div className="align-items-start d-flex">
-			{hasVirtualCluster && (
+			{hasVirtualCluster(activationKey) && (
```

Comment:
Try to transform in a variable before and use it to avoid call the function multiple times.
```suggestion
			{hasVirtualClusterForActivationKey && (
```